stdpopsim/catalog/EscCol/species.py

```diff
@@ -32,8 +32,7 @@
             # Wielgoss et al. (2011) calculated for strain REL606,
             # from synonymous substitutions over 40,000 generations.
             mutation_rate=8.9e-11,
-            recombination_rate=0.0,
-            gene_conversion_rate=8.9e-11,
+            recombination_rate=8.9e-11,
```

Comment:
There is currently no citation associated with recombination rate in E. Coli. From discussion with @fbaumdicker , I understand that this can be attributed to Wielgoss + [Didelot et al. (2012) ](https://pubmed.ncbi.nlm.nih.gov/22712577/)